### PR TITLE
Update Governance page

### PR DESCRIPTION
**Pull request by mbauman**
The goal here is to clarify the status quo a bit better:

* Make it a little less conversational
* Note the triage call
* Note the triage permissions on JuliaLang/julia repo
* ~~Note community communication channel moderation and administration~~
* Remove community manager as the position is currently vacant

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+One particularly important team is that of the "committers": these 40-some individuals have commit privileges and work together to maintain the repositories, including [`julia`](https://github.com/JuliaLang/julia) itself.
+This set of people doesn’t really define the project, however, since there are many people who are prolific contributors to the Julia ecosystem but who do not have this “commit bit.”
+An informal consensus process is used to review and merge pull requests, and a standing meeting — the [triage call](/community/working-groups/#triage_call) — aims to provide guidance and help move contentious issues forward.
 
-The Julia Project also employs a Community Manager. The role began with [Alex Arslan in 2017](https://discourse.julialang.org/t/welcome-alex-as-our-julia-community-manager/3756) and it is currently filled by [Logan Kilpatrick](https://twitter.com/OfficialLoganK) who started in early 2020.
+A larger set of individuals participates as "triage" members of the JuliaLang GitHub organization; they have permissions to organize and label the issues and pull reuqests.
```
Comment (mbauman) [inline governance.md:17]:
```suggestion
A larger set of individuals participates as "triage" members of the JuliaLang GitHub organization; they have permissions to organize and label the issues and pull requests.
```

**Review by TheCedarPrince — APPROVED** [review]
Know you weren't requesting a review @mbauman , but this looks great to me and very clear.

File: governance.md [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
+An informal consensus process is used to review and merge pull requests, and a standing meeting — the [triage call](/community/working-groups/#triage_call) — aims to provide guidance and help move contentious issues forward.
 
-The Julia Project also employs a Community Manager. The role began with [Alex Arslan in 2017](https://discourse.julialang.org/t/welcome-alex-as-our-julia-community-manager/3756) and it is currently filled by [Logan Kilpatrick](https://twitter.com/OfficialLoganK) who started in early 2020.
+A larger set of individuals participates as "triage" members of the JuliaLang GitHub organization; they have permissions to organize and label the issues and pull requests.
+
+## Community Communication Channels
+
+A number of [official community channels](/community/#community_channels) provide venues for discussion, collaboration, and media. Each is managed by an independent set of administrators and moderators whose role is largely to ensure a safe and welcoming environment, with all participants respecting the [Julia Community Standards](https://julialang.org/community/standards/).
```
Comment (vtjnash) [inline governance.md:21]: "A number of" could be replaced by the word "Several". I think "independent" here could use some clarification (eg independent of what?). Perhaps saying "moderators have been recognized in each channel for their contributions and willingness to do that work, but are not necessarily the same people across the various platforms" or something to that effect?

**Review by vchuravy — APPROVED** [review]

File: governance.md (hunk 2 above)
Comment (mbauman) [reply governance.md:21]: Yeah, my aim was to very roughly state that the moderators and administrators across the "community communication channels" are not necessarily the same, but they all have the same goal.  But, looking back at this, perhaps we're not quite ready to add these to the governance page as we don't currently make a distinction between community-controlled-and-moderated (discourse/slack/zulip/discord/youtube-channel) and other platforms the Julia community uses but has less (or no) Julia-community-owned administrative/moderation control (StackOverflow/Twitter).

Let's pull this section back for the time being.

Comment (mbauman) [inline governance.md:21]:
```suggestion
```

**Comment by ViralBShah** [comment]
Merge?

**Pull request merged**